.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "node"
+script:
+  - npm run lint
+  - npm test
```

Comment:
do we also want to make sure `npm run build` works? or perhaps `npm run build:test`